app/models/host/managed.rb

```diff
@@ -395,7 +395,7 @@ def info
     end
     if SETTINGS[:unattended]
       param["root_pw"]      = root_pass unless (!operatingsystem.nil? && operatingsystem.password_hash == 'Base64')
-      param["puppet_ca"]    = puppet_ca_server if puppetca?
+      param["puppet_ca"]    = puppet_ca_server if puppetca_exists? 
```

Comment:
rubocop is highlighting the trailing space on this line as a warning, please delete it.